recipes/cctbx/fix_macos_rpath.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tool for initial rpath fix for extensions
+"""
+from __future__ import absolute_import, division, print_function
+
+import os
+import glob
+
+from subprocess import check_output
+
+# =============================================================================
+if __name__ == '__main__':
+  exe_dev_files = glob.glob('build/exe_dev/*')
+  ext_files = glob.glob('build/lib/*_ext.so')
+  lib_files = glob.glob('build/lib/*.dylib')
+  test_files = glob.glob('build/**/tst_*', recursive=True) \
+               + glob.glob('build/**/timing/*', recursive=True) \
+               + glob.glob('build/**/boost_python/*.so', recursive=True)
+  for ext_file in exe_dev_files + ext_files + lib_files + test_files:
+    libraries = check_output(['otool', '-L', ext_file]).decode('utf8').split('\n')
```

Comment:
I think this should use `$OTOOL` and `$INSTALL_NAME_TOOL` as the conda-forge provided binaries are prefixed (last I checked it was `x86_64-apple-darwin13.4.0-install_name_tool`/`x86_64-apple-darwin13.4.0-otool` but you shouldn't rely on a fixed value).

```suggestion
    libraries = check_output([os.environ['OTOOL'], '-L', ext_file]).decode('utf8').split('\n')
```